RetailCoder.VBE/Inspections/QuickFixes/PassParameterByReferenceQuickFix.cs

```diff
@@ -15,11 +15,13 @@ namespace Rubberduck.Inspections.QuickFixes
     public class PassParameterByReferenceQuickFix : QuickFixBase
     {
         private Declaration _target;
+        private AssignedByValParameterQuickFixHelper _quickFixHlpr;
```

Comment:
I've used the helper class within PassParameterByReferenceQuickFix because this QuickFix is only used by the AssignedByValParameterInspection.  Another alternative is to rename the two related QuickFix classes to: AssignedByValParameterMakeLocalCopyQuickFix and AssignedByValParameterChangeToByRefQuickFix.  This would be more precise, and then the helper class name makes a little more sense.  Going to the more generic QuickFixHelper is also OK, but it implies that the class is will need to meet a bit higher standard as generic helper.